### ISSUE TITLE
clicking on "JUST TAKEN" starts the camera app on Pixel 7 (and more)

### ISSUE THREAD

**Issue by andrearicci** [CorruptedArk/did-i-take-my-meds#89]
**Describe the bug**
2 bugs:
1-On my wife's PIXEL 7  notification there's not the option "taken" but only "remember me in 15 minutes". 
2- she clicks opening the app where she finds the big JUST TAKEN button. But when she clicks on it the camera app pops up. 

**To Reproduce**
1. Go to the app trough a notification
2. Click on JUST TAKEN
3. Camera app starts, the medicine is not recorded as taken

**Expected behavior**

**Screenshots**

**Smartphone (please complete the following information):**
 - Device: Pixel 7
 - OS: Android 14
 - Version: 14
 - Build Location: F-Droid release

**Additional context**

**Comment by akovia** [comment]
Same exact behavior with some minor differences in hardware and software. I've attempted to turn off the camera permission, but it still launches which I find odd.

Still haven't figured out how to log a taken medication.

- Device: Pixel 8 Pro
- OS: Android 15
- Version: 1.6.6
- Build Location: GitHub release

**Comment by eepykate** [comment]
isnt this a feature?  when making a tracker, disable "require photo proof"

**Comment by andrearicci** [comment]
yes it is a feature, sorry for opening this: it's on by default and you can turn it off
